### DISCUSSION TITLE
How to implement many-to-one RNN (MXnet + Python)?

### DISCUSSION THREAD

**Discussion by nikudyshko** [apache/incubator-mxnet#19741] [Q&A]
As a title says, maybe someone could give some information/advises on how to do this?

**Comment by szha** [comment]
The simplest approach is to use the rnn layer in Gluon. See https://mxnet.apache.org/versions/1.7.0/api/python/docs/api/gluon/rnn/index.html#mxnet.gluon.rnn.LSTM.

The example attached in that API returns the outputs from every time step, and from that you can slice and get the output for one time-step and use that for later modeling (e.g. a dense layer as a classifier for sequence classification)
